native/explorer/src/series.rs

```diff
@@ -1215,6 +1215,24 @@ pub fn s_to_time(s: ExSeries) -> Result<ExSeries, ExplorerError> {
     Ok(ExSeries::new(s1))
 }
 
+#[rustler::nif(schedule = "DirtyCpu")]
+pub fn s_date_diff(data: ExSeries, other: ExSeries) -> Result<ExSeries, ExplorerError> {
+    let s = data.clone_inner();
+    let s1 = other.clone_inner();
+
+    let res = (s - s1)
+        .duration()?
+        .into_iter()
+        .map(|d| match d {
+            Some(d) => Some(d / 86400000),
+            None => None,
+        })
+        .collect::<Int64Chunked>()
```

Comment:
Do you need to iterate? I believe you should be able to call `/ 86400000` or similar on a series to compute the division.